data-browser/src/components/Parent.tsx

```diff
@@ -39,31 +63,85 @@ type NestedParentProps = {
 function NestedParent({ subject, depth }: NestedParentProps): JSX.Element {
   const resource = useResource(subject, { allowIncomplete: true });
   const [parent] = useString(resource, properties.parent);
-
+  const navigate = useNavigate();
+  const [title] = useTitle(resource);
   depth = depth + 1;
 
   // Prevent infinite recursion, set a limit to parent breadcrumbs
   if (depth > 5) {
```

Comment:
Extracting this (magic) number to a constant (eg. MAX_BREADCRUM_AMOUNT) makes the code speak for itself and probably removes the need for the comment.
```ts
const MAX_BREADCRUM_AMOUNT = 4;
...
if (depth > MAX_BREADCRUM_AMOUNT) {
  ...
}
```